pq-src/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pq-src"
-version = "0.3.6+libpq-17.4"
+version = "0.3.7+libpq-17.7"
```

Comment:
@weiznich You mention libpq-17.7 here, but libqp 17.5 in the CHANGELOG.md, is that intentional?

Reply:
Seems like I managed to screw up that version. I've yanked the 0.3.7 release and pushed a 0.3.8 release with the correct version specifier.  https://github.com/sgrif/pq-sys/releases/tag/v0.3.8

Thanks for noticing.